agents-common/src/test/java/org/apache/ranger/plugin/resourcematcher/RangerURLResourceMatcherTest.java

```diff
@@ -33,120 +33,120 @@
 import static org.junit.Assert.assertEquals;
 
 public class RangerURLResourceMatcherTest {
-
     Object[][] data = {
             // { resource, policy, optWildcard, recursive, result
-            { "hdfs://hostname:8020/app/warehouse/data/emp.db",    "hdfs://hostname:8020/app/warehouse/*", true, true, true, "user" },
```

Comment:
Is this suggested by the checkstyle ? I believe earlier was more readable.

Reply:
I agree. This change likely was done by IDE code reformatting. Updated.